userspace/falco/configuration.cpp

```diff
@@ -90,6 +90,13 @@ void falco_configuration::init(const std::vector<std::string>& cmdline_options)
 	load_yaml("default");
 }
 
+void falco_configuration::init_from_content(const std::string& config_content, const std::vector<std::string>& cmdline_options)
```

Comment:
I'd rather add the `config_content=""` optional argument to the old `init` method (the one above) instead of adding a completely new method.

Reply:
I think they are both just used by tests anyway.

Reply:
I was a bit nervous about adding an optional argument that completely changes the method behavior. Perhaps to make it cleaner I could split that into `init_from_file()` and `init_from_content()`, wdyt?

Reply:
>  init_from_file() and init_from_content()

Like it!

Reply:
Done! :)